.github/workflows/benchmark.yml

```diff
@@ -141,11 +141,16 @@ jobs:
         env:
           THEME: ${{ inputs.theme }}
 
+      # See https://issues.chromium.org/issues/373753919
+      # and https://chromium.googlesource.com/chromium/src/+/main/docs/security/apparmor-userns-restrictions.md
+      - name: Disable AppArmor
+        run: echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns
+
       - name: Benchmark Web Vitals
         working-directory: wpp-research
         run: |
-          npm run research --silent -- benchmark-web-vitals -u http://localhost:8881/ -n 100 -p -v -o csv > before.csv
-          npm run research --silent -- benchmark-web-vitals -u http://localhost:8891/ -n 100 -p -v -o csv > after.csv
+          npm run research --silent -- benchmark-web-vitals -u http://localhost:8881/ -n 100 -c "Fast 4G" -p -v -o csv > before.csv
+          npm run research --silent -- benchmark-web-vitals -u http://localhost:8891/ -n 100 -c "Fast 4G" -p -v -o csv > after.csv
```

Comment:
What about adding `-e "Moto G4"` these as well? Otherwise, the tests are done with a `960x700` viewport size which doesn't really correspond with commonly used devices.

Reply:
Why not test a desktop viewport?

For example the core performance tests only run for desktop right now, so if we test mobile here then it won't be easy to reproduce the results in core.

Reply:
OK, so then `-w desktop` would use a better desktop viewport size.

I was thinking mobile since optimization has a bigger impact there, but I wasn't thinking about the core performance tests.

But if the core performance tests are using `960x700` viewport size, then this may not be a good suggestion anyway.

Reply:
Core uses the [Desktop Chrome](https://github.com/microsoft/playwright/blob/5fce12dcca2ef518bbb91292b5d8e4989097a0ed/packages/playwright-core/src/server/deviceDescriptorsSource.json#L1624-L1638) preset from Playwright, which uses a 1280x720 viewport

Reply:
Great. Just double-checking for myself:

https://github.com/GoogleChromeLabs/wpp-research/blob/68af1efbcc4657c6492e66a7fbbabbb606c3aa69/cli/commands/benchmark-web-vitals.mjs#L195-L197

Namely, where `{ width: 960, height: 700 }` is supposed to be `Viewport similar to @wordpress/e2e-test-utils 'large' configuration.`

Indeed, this is still the case:

https://github.com/WordPress/gutenberg/blob/400ee91f4a0ee80ba1788a634fe9c8feb833bfb4/packages/e2e-test-utils/src/set-browser-viewport.js#L28-L29

Reply:
Note that WP doesn‘t use that package anymore